examples/tracing/otlp-collector/otlp-collector.ipynb

```diff
@@ -0,0 +1,135 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
```

Comment:
add some description at the head of the notebook describe the purpose and what user can gain from this